test/unit/presenters/search_result_test.rb

```diff
@@ -2,30 +2,8 @@
 require "test_helper"
 
 class SearchResultTest < ActiveSupport::TestCase
-  should "display a description" do
-    result = SearchResult.new(SearchParameters.new({}), "description" => "I like pie")
-    assert_equal "I like pie", result.description
-  end
-
-  should "not display a generic description for other formats which are missing them" do
-    result = SearchResult.new(SearchParameters.new({}), "format" => "edition", "title" => "VAT")
-    assert_nil result.description
-  end
-
-  should "truncate descriptions to a maximum of 215 characters" do
-    result = SearchResult.new(SearchParameters.new({}),
-                              "description" => "Long description is long "*100)
-    assert(result.description.length <= 215)
-  end
-
-  should "end the description with ellipsis if truncated" do
-    result = SearchResult.new(SearchParameters.new({}),
-                              "description" => "Long description is long "*100)
-    assert result.description.end_with?('…')
-  end
-
   should "report when no examples are present" do
-    result = SearchResult.new(SearchParameters.new({}), "description" => "I like pie").to_hash
+    result = SearchResult.new(SearchParameters.new({}), {}).to_hash
```

Comment:
The second argument is the search_result. We're now just passing in an empty result, which is enough to make the tests pass. I suppose this hash used to contain a value for `description` because we we modified the description a lot and didn't want it to be nil.